MediaGalleryUi/etc/db_schema.xml

```diff
@@ -13,6 +13,7 @@
         <column xsi:type="varchar" name="preview_url" length="255" nullable="true" comment="Preview Url"/>
         <column xsi:type="int" name="width" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Width"/>
         <column xsi:type="int" name="height" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Height"/>
+        <column xsi:type="datetime" name="updated_at" nullable="false" default="CURRENT_TIMESTAMP" comment="Modification Date"/>
```

Comment:
That is an index table, the default value `CURRENT_TIMESTAMP` might be incorrect, it should not have the default value and should be populated from media_gallery_asset only